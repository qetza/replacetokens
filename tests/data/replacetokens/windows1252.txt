windows1252
€
#{var1}#