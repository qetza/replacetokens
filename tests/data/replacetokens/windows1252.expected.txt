windows1252
€
var1_value